lib/daru/date_time/index.rb

```diff
@@ -366,6 +366,24 @@ def [] *key
       end
     end
 
+    def pos *args
+      # to filled
+      out = self[*args]
+      return out if out.is_a? Numeric
+      out.map { |date| self[date] }
+    end
+
+    def subset *args
+      self[*args]
+    end
+
+    def respond? *args
+      self[*args]
+      true
+    rescue IndexError
```

Comment:
see http://stackoverflow.com/questions/1542672/how-does-one-use-rescue-in-ruby-without-the-begin-and-end-block